python/lsst/multiprofit/asinhstretchsigned.py

```diff
@@ -59,7 +58,7 @@ class AsinhStretchSigned(apvis.BaseStretch):
         to logarithmic behavior, expressed as a fraction of the
         normalized image.  Must be in the range between 0 and 1.
         Default is 0.1
-    """
+    """  # noqa: W505
```

Comment:
Is this because of the math? Can you put a backslash in after the `/` to fix that? Does it render okay in sphinx?

Reply:
The math line is 10 characters too long, yes. Is a backslash supposed to split it? Anyway, most of this is copy-pasted from astropy and lightly modified. I haven't actually configured docs for the package yet so I'll do that on another ticket.